packages/pwa-buildpack/lib/cli/create-project.js

```diff
@@ -0,0 +1,215 @@
+            return packageRoot;
+        }
+    } catch (e) {
+        // Not cached.
+    }
+    let tarballUrl;
+    try {
+        prettyLogger.info(`Finding ${packageName} tarball on NPM`);
+        tarballUrl = JSON.parse(
+            execa.shellSync(`npm view --json ${packageName}`, {
+                encoding: 'utf-8'
+            }).stdout
+        ).dist.tarball;
+    } catch (e) {
+        throw new Error(
+            `Invalid template: could not get tarball url from npm: ${e.message}`
+        );
+    }
+
+    let tarballStream;
+    try {
+        prettyLogger.info(`Downloading and unpacking ${tarballUrl}`);
+        tarballStream = (await fetch(tarballUrl)).body;
+    } catch (e) {
+        throw new Error(
+            `Invalid template: could not download tarball from NPM: ${
+                e.message
+            }`
+        );
+    }
+
+    await fse.ensureDir(packageDir);
+    return new Promise((res, rej) => {
+        const untarStream = tar.extract({
+            cwd: packageDir
+        });
+        tarballStream.pipe(untarStream);
+        untarStream.on('finish', () => {
+            prettyLogger.info(`Unpacked ${packageName}`);
+            res(packageRoot);
+        });
+        untarStream.on('error', rej);
+        tarballStream.on('error', rej);
+    });
+}
+
+async function findTemplateDir(templateName) {
+    const template = templateAliases[templateName] || {
+        npm: templateName,
+        dir: templateName
+    };
+    try {
+        await fse.readdir(template.dir);
+        prettyLogger.info(`Found ${templateName} directory`);
+        // if that succeeded, then...
+        return template.dir;
+    } catch (e) {
+        prettyLogger.info(`No template at ${template.dir}.`);
+        return makeDirFromNpmPackage(template.npm);
+    }
+}
+
+module.exports.sampleBackends = require('../../sampleBackends.json');
+
+module.exports.command = 'create-project <directory>';
+
+module.exports.describe =
+    'Create a PWA project in <directory> based on template.';
+
+module.exports.builder = yargs =>
+    yargs
+        .version()
+        .showHelpOnFail(false)
+        .positional('directory', {
+            describe:
+                'Name or path to a directory to create and fill with the project files. This directory will be the project root.',
+            normalize: true
+        })
+        .group(['template', 'backendUrl'], 'Project configuration:')
+        .options({
+            template: {
+                describe:
+                    'Name of a "template" to clone and customize. Currently only the "venia-concept" template is supported: `buildpack create-project --template venia-concept`',
+                choices: ['venia-concept']
+            },
+            backendUrl: {
+                alias: 'b',
+                describe:
+                    'URL of the Magento 2.3 instance to use as a backend. Will be added to `.env` file.'
+            }
+        })
+        // .demandOption('template')
```

Comment:
We don't need this if template option is a list of choices, so I think we can remove this comment entirely.